index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
```

Comment:
This is ... clever. I've not seen this solution used before. Unfortunately, it's also invalid markup (can't have a raw input floating out in the body of the document). I'd recommend spending some additional time looking at the Zurb sidebar or [this Smashing Magazine Article](http://www.smashingmagazine.com/2013/01/off-canvas-navigation-for-responsive-website/) for slightly more semantic ways of solving this.

Off-canvas navigation is somewhat tricky, especially since it will generally add more markup to your document. That being said, it's possible without going crazy.